frontend/src/Student/components/Portfolios.js

```diff
@@ -13,41 +13,57 @@ const NoShowsContainer = styled.div`
   transform: translate(-50%, -50%);
 `
 class Portfolios extends Component {
-  renderShows = shows => {
-    if (shows.length === 0) {
+  renderPortfolios = (portfolios, openPeriod) => {
+    if(openPeriod){
+      const skeletonPortfolio = {
+        portfolioPeriod: openPeriod,
+        pieces: [],
+        id: -1
+      }
+
+      if(!portfolios){
+        portfolios = []
+      }
+      portfolios = [skeletonPortfolio].concat(portfolios)
```

Comment:
This is adding a fake portfolio around the open portfolio period object so that it can be displayed to the student before they have made a portfolio for it. The reason this does a `concat` (instead of an `unshift` is because the `portfolios` array is immutable.